include/nbl/asset/CCPUMeshPackerV2.h

```diff
@@ -202,12 +203,12 @@ bool CCPUMeshPackerV2<MDIStructType>::commit(IMeshPackerBase::PackedMeshBufferDa
 }
 
 template <typename MDIStructType>
-typename CCPUMeshPackerV2<MDIStructType>::PackerDataStore CCPUMeshPackerV2<MDIStructType>::createGPUPackerDataStore(video::IVideoDriver* driver)
+typename CCPUMeshPackerV2<MDIStructType>::PackerDataStore<video::IGPUBuffer> CCPUMeshPackerV2<MDIStructType>::createGPUPackerDataStore(video::IVideoDriver* driver)
```

Comment:
wgt actually mesh packer needs `MDIStructType` template param?

Reply:
because the GPU driver supports having variable strides between the MDI structs (you can put whatever you want after the 20th byte).

You want to give power to the user.

Not sure now that template is the best way to achieve that, but this is what I advised at the time.

Reply:
ah, ok, nice